contracts/src/HypercertMinter.sol

```diff
@@ -201,8 +212,7 @@ contract HypercertMinter is IHypercertToken, SemiFungible1155, AllowlistMinter,
      * variables without shifting down storage in the inheritance chain.
      * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
      * Assuming 30 available slots (slots cost space, cost gas)
-     * 1. name
-     * 2. typeRestrictions
+     * 1. typeRestrictions
      */
-    uint256[28] private __gap;
+    uint256[29] private __gap;
```

Comment:
I'm not sure I understand what's going on here....  I'm not sure that I see storage has changed in this contract. Mind sharing the thought process real quick?

Reply:
Not al variable are `storage`. Immutables and constants are set once, in code or constructor respectively, and not assigned storage slots.